pwiz_tools/Skyline/Model/Irt/RCalcIrt.cs

```diff
@@ -243,6 +244,38 @@ public IEnumerable<DbIrtPeptide> GetDbIrtPeptides()
             IEnumerable<IRetentionTimeProvider> providers, int countProviders,
             DbIrtPeptide[] standardPeptideList, DbIrtPeptide[] items)
         {
+            var matchedStandard = IrtStandard.WhichStandard(standardPeptideList.Select(pep => pep.ModifiedTarget));
+            if (matchedStandard != null)
+            {
+                var dummyDoc = new SrmDocument(SrmSettingsList.GetDefault());
+                using (var reader = matchedStandard.DocumentReader)
+                {
+                    if (reader != null)
+                    {
+                        dummyDoc = dummyDoc.ImportDocumentXml(reader,
+                            string.Empty,
+                            MeasuredResults.MergeAction.remove,
+                            false,
+                            null,
+                            Settings.Default.StaticModList,
+                            Settings.Default.HeavyModList,
+                            null,
+                            out _,
+                            out _,
+                            false);
+                        standardPeptideList = standardPeptideList.Select(pep => new DbIrtPeptide(pep)).ToArray();

```

Comment:
You end up replacing every element in this list, so you would probably just start with an empty list.